src/overlays/actors/ovl_En_Bigokuta/z_en_bigokuta.c

```diff
@@ -52,104 +67,834 @@ static ColliderJntSphInit D_809BF3F8 = {
+    Camera_ChangeSetting(globalCtx->cameraPtrs[MAIN_CAM], 0x34);
+    func_8005AD1C(globalCtx->cameraPtrs[MAIN_CAM], 4);
+
+    if (this->cylinder[0].base.atFlags & AT_ON) {
+        if (this->actionFunc != func_809BE058) {
+            for (i = 0; i < ARRAY_COUNT(this->cylinder); i++) {
+                CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->cylinder[i].base);
+            }
+            this->actor.flags |= 0x1000000;
+        } else {
+            for (i = 0; i < ARRAY_COUNT(this->cylinder); i++) {
+                CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->cylinder[i].base);
+            }
+        }
+        for (i = 0; i < ARRAY_COUNT(this->cylinder); i++) {
+            CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->cylinder[i].base);
+        }
+        if (this->collider.base.acFlags & AC_ON) {
+            CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+        }
+    }
+    if (this->collider.base.acFlags & AC_ON) {
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    } else {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    }
+    Actor_SetFocus(&this->actor, this->actor.scale.y * 25.0f * 100.0f);
+    func_809BCEBC(this, globalCtx);
+}
+
+s32 EnBigokuta_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                                void* thisx) {
+    EnBigokuta* this = THIS;
+    u8 v0;
+    f32 temp_f0;
+    s32 temp_hi;
+
+    if (limbIndex == 15) {
+        if (this->actionFunc == func_809BE058 || this->actionFunc == func_809BE180) {
+            OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_bigokuta.c", 1914);
+
+            if (this->actionFunc == func_809BE058) {
+                temp_hi = this->unk_196 % 12;
+                if (temp_hi >= 8) {
+                    temp_f0 = (12 - temp_hi) * (M_PI / 8);
+                } else {
+                    temp_f0 = temp_hi * (M_PI / 16);
+                }
+            } else {
+                temp_hi = this->unk_196 % 6;
+                if (temp_hi >= 4) {
+                    temp_f0 = (6 - temp_hi) * (M_PI / 4);
+                } else {
+                    temp_f0 = temp_hi * (M_PI / 8);
+                }
+            }
+
+            temp_f0 = sinf(temp_f0) * 125.0f + 130.0f;
+            gDPPipeSync(POLY_OPA_DISP++);
+
+            gDPSetEnvColor(POLY_OPA_DISP++, temp_f0, temp_f0, temp_f0, 255);
+            CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_en_bigokuta.c", 1945);
+        }
+    } else if (limbIndex == 10) {
+        OPEN_DISPS(globalCtx->state.gfxCtx, "../z_en_bigokuta.c", 1950);
+        if (this->actionFunc == func_809BE26C) {
+            v0 = this->unk_196 * (255.0f / 38);
+        } else {
+            v0 = 255;
+        }
+        gDPPipeSync(POLY_OPA_DISP++);
+        gDPSetEnvColor(POLY_OPA_DISP++, v0, v0, v0, v0);
```

Comment:
I feel like I've seen this named `intensity` in other contexts.  I think in TitleCard_Draw?